kratos/python_scripts/analysis_stage.py

```diff
@@ -344,6 +349,30 @@ def __CreateListOfProcesses(self):
         self._list_of_output_processes = self._CreateProcesses("output_processes", order_processes_initialization)
         self._list_of_processes.extend(self._list_of_output_processes) # Adding the output processes to the regular processes
 
+    def _GetMapOfAdditionalHistoricalVariables(self):
+        """This function returns the list of (additional) historicsl variables
+        needed in the simulation
+        It can be overridden in derived classes

```

Comment:
can you update and improve a bit the docs please?

Reply:
done!